### ISSUE TITLE
Ellipsis mixin creating inline-block phantom space in Chrome/Firefox.

### ISSUE THREAD

**Issue by cloud-walker** [styled-components/polished#256]
- `polished` version: 1.8
- `styled-components` library and version:  2.2.1
- Any relevant JS library and version: React 16

Reproduction: https://codesandbox.io/s/9j32z5z7p

**Comment by bhough** [comment]
@cloud-walker what browser/version are you seeing this in? Here is what I see in the latest Chrome: http://d.pr/i/Gk93KN

**Comment by cloud-walker** [comment]
I see the same exact thing, try to add a border to the `DropdownItemBroken`

The broken one has an "invisible" margin on top and bottom.

Ref: https://css-tricks.com/fighting-the-space-between-inline-block-elements/

**Comment by bhough** [comment]
@cloud-walker I will have to do some testing on this. Some browsers this works fine in (such as Safari), others treat the ellipsis as a space. `inline-block` is part of the original Bourbon mixin this was ported over from, so I hesitate to change it without seeing what impact it has on other browsers first.

**Comment by cloud-walker** [comment]
I think there is no clear solution to this.

Avoid using `display: inline-block` exposes the mixin to external implicit display (such span, div, etc).

`display: inline-flex` fix this, but is not backwards compatible

**Comment by bhough** [comment]
Agreed, after digging into this further, the inline-block causes more good than harm (especially, since it is inconsistent across browsers). Feel better about leaving it at this point, then modifying it.

**Comment by cloud-walker** [comment]
Then what do you think about a combination of `inline-block` + `inline-flex`?

Like that:

```css
.foo {
  /* ellipsis stuff */
  display: inline-block;
  display: inline-flex;
}
```

Older browsers will fallback to the inline-block solution!

**Comment by bhough** [comment]
Addressed in 2.0: https://github.com/styled-components/polished/milestone/1
